content/manuals/accounts/deactivate-user-account.md

```diff
@@ -7,22 +7,23 @@ keywords: Docker Hub, delete, deactivate, account, account management
 
 You can deactivate an account at any time. This section describes the prerequisites and steps to deactivate a user account. For information on deactivating an organization, see [Deactivating an organization](../admin/deactivate-account.md).
 
->**Warning**
+>[!Warning]
 >
 > All Docker products and services that use your Docker account will be inaccessible after deactivating your account.
 
 ## Prerequisites
 
 Before deactivating your Docker account, ensure that you meet the following requirements:
 
-- You must not be a member of a company or any teams or organizations. You must remove the account from all teams, organizations, or the company.
-
+- For owners, you must leave your organization or company before deactivating your Docker account.
     To do this:
-    1. Navigate to **Organizations** and then select the organization(s) you need to leave.
-    2. Find your username in the **Members** tab.
-    3. Select the **More options** menu and then select **Leave organization**.
+    1. Sign in to [Docker Home](https://app.docker.com/).
```

Comment:
Not necessary, but maybe combine steps 1 and 2 for a shortcut. `Sign in to the [Docker Admin Console](https://app.docker.com/admin)`.